src/invest_reports/utils.py

```diff
@@ -174,6 +177,7 @@ def plot_raster_list(tif_list, datatype_list, transform_list=None):
             imshow_kwargs['norm'] = transform
         if dtype == 'binary':
             transform = matplotlib.colors.BoundaryNorm([0, 0.5, 1], cmap.N)
+            # @TODO: ¿update imshow_kwargs['norm']?
```

Comment:
I guess we always want a linear normalization for a binary raster, so we could be explicit about that here, overriding whatever the user passed for `transform` just in case it was something other than "linear" or nothing. I still think these functions and the constants we're using for `transform` and `dtype` are worth revisiting to make sure we are happy with them before asking our colleagues to use them. I'm happy to take that on.